pyvista/plotting/renderer.py

```diff
@@ -695,6 +695,32 @@ def has_charts(self):  # numpydoc ignore=RT01
         """Return whether this renderer has charts."""
         return self._charts is not None and len(self._charts) > 0
 
+    @property
+    def charts(self):  # numpydoc ignore=RT01
+        """Return a list of all charts in this renderer.
+
+        Examples
+        --------
+        .. pyvista-plot::
+           :force_static:
+
+           >>> import matplotlib.pyplot as plt
+           >>> import pyvista as pv
+           >>> fig, ax = plt.subplots()
+           >>> _ = ax.plot([1, 2, 3], [4, 5, 6])
+           >>> chart = pv.ChartMPL(fig)
+           >>> pl = pv.Plotter(shape=(1, 2))
+           >>> pl.subplot(0, 0)
+           >>> _ = pl.add_mesh(pv.Sphere())
+           >>> pl.subplot(0, 1)
+           >>> _ = pl.add_chart(chart)
+           >>> ax = pl.renderer.charts[0].figure.axes[0]
+           >>> ax.get_lines()[0].set_ydata([6, 5, 4])
+           >>> pl.show()
+
+        """
+        return self._charts
```

Comment:
We need to check that the renderer has charts and otherwise return an empty list. I would also just return a (new) list of charts instead of the whole `Charts` object, which was meant for internal usage.
E.g.
```suggestion
        return [*self._charts] if self.has_charts else []
```